docs/DeveloperGuide.md

```diff
@@ -838,9 +838,32 @@ Some general utility features
 
 #### 5.3.1. Clear transactions
 
-#### 5.3.2. Help Window
+#### 5.3.2. Help 
 
-#### 5.3.3. etc
+##### Overview
+
+The `help` command can be used in 2 different ways.
+
+1. `help` with no arguments
+2. `help <command_word>` with 1 argument containing a command word.
+
+##### Details
+
+1. `help`
+
+Opens a Help Window with a link to our User Guide. 
+Displays a help message on the right panel containing a list of all our commands.
```

Comment:
Also, if you're planning to have these appear on new lines, should be 

```markdown
Opens...

Displays...
```


The current formatting will just display everything on the same line

Reply:
Yeap, supposed to be 1 line, I have clarified it.